frontend/lib/src/components/widgets/DataFrame/hooks/useDataLoader.ts

```diff
@@ -48,6 +49,12 @@ function useDataLoader(
 ): DataLoaderReturn {
   const getCellContent = React.useCallback(
     ([col, row]: readonly [number, number]): GridCell => {
+      // Special case for empty dataframes
+      if (numRows === 0) {
+        // Return an empty cell for empty dataframes
+        return getEmptyCell()
+      }
```

Comment:
Solving this using the `ignoredRowIndices` parameter of `useTooltips` would be a bit more consistent. 
The `getCellContent` is only supposed to be called for actual data cells; everything else is expected to return an error since it's an unexpected situation. The `empty` row of a dataframe with 0 rows isn't part of the actual data, it's just a visual gimmick we add that shouldn't cause the tooltip implementation to trigger a getCellContent call. It is similar to the trailing row that is added in the data editor, which is ignored via the `ignoredRowIndices` parameter. In this case, we can just do something like: `isEmptyTable ? [0] : isDynamicAndEditable ? [numRows] : []`